### PR TITLE
Plan for Pod Security Standard and Validation Failure Action

### PR DESCRIPTION
**Pull request by fjogeleit**
Thank you for wanting to submit a Pull Request to the Civo Kubernetes Marketplace repository!

If your pull request concerns an existing Marketplace application, please make sure you have:

* [x] Notified the Maintainer of the application in this pull request so that they are aware of your proposal.

@JimBugwadia 

* [x] Outlined the changes you are proposing, and the reasons these are required (e.g. stability, compatibility with new versions of Kubernetes implementations, etc).

#198 New Plan to make it possible to choose between the Default and Restricted Pod Security Standard and the audit or enforce mode for this policies.

* [x] Tested that the application works with the proposed updates applied (including a screenshot)

Bildschirmfoto 2021-05-13 um 19 41 43

### CODE REVIEW

File: kyverno/manifest.yaml [hunk 1]
```diff
@@ -6,3 +6,20 @@ maintainer: "Kyverno Authors"
 description: Kyverno is a policy engine designed for Kubernetes
 url: https://kyverno.io
 category: security
+plans:
+  - label: "Default Policy Security Standard in audit mode"
```
Comment (JimBugwadia) [inline kyverno/manifest.yaml:10]: should be `Policy Security Standards` (missing `s`) everywhere

File: kyverno/manifest.yaml [hunk 2]
```diff
@@ -6,3 +6,20 @@ maintainer: "Kyverno Authors"
...
+    configuration:
+      POD_SECURITY_STANDARD:
```
Comment (JimBugwadia) [inline kyverno/manifest.yaml:12]: would be good to rename to `POD_SECURITY_STANDARDS`

**Review by JimBugwadia — CHANGES_REQUESTED** [review]
looks good! a minor change request.

File: kyverno/manifest.yaml (hunk 2 above)
Comment (fjogeleit) [reply kyverno/manifest.yaml]: changed

File: kyverno/manifest.yaml (hunk 1 above)
Comment (fjogeleit) [reply kyverno/manifest.yaml]: changed

**Comment by JimBugwadia** [comment]
Looks good to me! Thanks!

**Comment by saiyam1814** [comment]
@fjogeleit Can you make the Labels for the plan concise? 
because this is what you see in the UI. if you can create something like Linkerd!
image

**Comment by fjogeleit** [comment]
Hey @saiyam1814, thanks for your Feedback. I Updated the Labels

**Comment by saiyam1814** [comment]
LGTM, once this goes out in production please test (I will push to prod in sometime)

**Comment by fjogeleit** [comment]
Sure, thank you!

**Pull request merged**